docs/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>js-dpp - JavaScript implementation of the Dash Platform Protocol</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="keywords" content="dpp, dash, javascript, primitives, platform protocol, Identity, Data Contract">
+  <meta name="description" content="JavaScript implementation of the Dash Platform Protocol.">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css">
+</head>
+<body>
+<div id="app"></div>
+<script>
+  window.$docsify = {
+    alias: {
+      '.*?/': 'https://raw.githubusercontent.com/dashevo/js-dpp/master/README.md',
+    },
+    name: 'js-dpp',
+    repo: 'https://github.com/dashevo/js-dpp',
+    loadSidebar: "_sidebar.md",
+    maxLevel: 4,
+    subMaxLevel: 2,
+    search: {
+      noData: {
+        '/': 'No results!'
+      },
+      paths: 'auto',
+      placeholder: {
+        '/': 'Search'
+      }
+    },
+    // search: 'auto',
+    formatUpdated: '{MM}/{DD} {HH}:{mm}',
+    themeColor: '#008de4',
```

Comment:
I like to include this so it doesn't drop you in the middle of a page when you change pages.
```suggestion
    themeColor: '#008de4',
    auto2top: true,
```

Reply:
Neat !